python/README.md

```diff
@@ -1,23 +1,33 @@
-# Python Arrow Flight Client Application Example
-![Build Status](https://github.com/dremio-hub/arrow-flight-client-examples/workflows/python-build/badge.svg)
 
-This lightweight Python client application connects to the Dremio Arrow Flight server endpoint. Developers can use token based or regular user credentials (username/password) for authentication. Please note username/password is not supported for Dremio Cloud. Any datasets in Dremio that are accessible by the provided Dremio user can be queried. By default, the hostname is `localhost` and the port is `32010`. Developers can change these default settings by providing the hostname and port as arguments when running the client.
-Moreover, the tls option can be provided to establish an encrypted connection.
+## Getting Started
+1. Install [Python 3](https://www.python.org/downloads/)
+2. Download and install the [dremio-flight-endpoint whl file](https://github.com/dremio-hub/arrow-flight-client-examples/releases)
+    - `python -m pip install <PATH TO WHEEL>` 
+3. Clone this repo and changed your working directory to: arrow-flight-client-examples/python/  
+    - `git clone https://github.com/dremio-hub/arrow-flight-client-examples.git`  
+    - `cd arrow-flight-client-examples/python/`
+4. Run the Python sample application with a local instance of Dremio (with default parameters):
+    - `python3 example.py -username <USER> -password <password> -query 'SELECT 1'`
+
+## How to connect to Dremio Cloud
+
+Get started with your first query to Dremio Cloud.
 
-### Instructions on using this Python sample application
-- Install and setup Python3 as `pyarrow` requires Python3
-- This application also requires `pyarrow` and `pandas`. Consider one of the dependency installation methods below. We recommend using `conda` for its ease of use.
-- Install dependencies using `conda`
-  - `conda install -c conda-forge --file requirements.txt`
-- Alternatively, install dependencies using `pip` 
-  - `pip3 install -r requirements.txt`
-- Run the Python sample application with a local instance of Dremio (with default parameters):
-  - `python3 example.py -query 'SELECT 1'`
+* The following example requires you to create a [Personal Access Token](https://docs.dremio.com/software/security/personal-access-tokens/) in Dremio. Replace ```<INSERT PAT HERE>``` in the example below with your actual PAT token.
```

Comment:
The link here is for Dremio software - is there a different page for Dremio Cloud or is it just a single page?

Reply:
There's a separate link for cloud, but the instructions seem to be the same. I'll update it to cloud for the example.